src/parser/parser.yy

```diff
@@ -1169,6 +1169,9 @@ assignment_sentence
     : VARIABLE ASSIGN piped_sentence {
         $$ = new AssignmentSentence($1, $3);
     }
+    | VARIABLE ASSIGN set_sentence {
```

Comment:
piped_sentence is not one kind of set_sentence?

Reply:
No, it isn't.

Reply:
IMO we should do that.  @dutor WDYT?

Reply:
i'm ok with your opinion.